src/components/Calculator.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * /* eslint-disable react/prefer-stateless-function
+ *
+ * @format
+ */
```

Comment:
[OPTIONAL] Rather than disable the linter strive to fix them as this makes you more accustomed to the style guide of the tech.